src/monogame/include/csharp_GameWrapper.h

```diff
@@ -0,0 +1,20 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+                
+//
+// This code is auto generated, don't modify it by hand.
+// To regenerate it run: Tools/MonoGame.Generator.CTypes
+//
+
+#pragma once
+
+#include "csharp_common.h"
```

Comment:
The prefix `csharp_` feels wrong here.  Better ideas?  `gen_` or `api_` or something?

Reply:
```api_``` or ```export_``` seems more fitting but ```csharp_``` is fine enough to me. Nitpicking here.